docs/sphinx/conf.py

```diff
@@ -24,7 +24,7 @@
 # -- General configuration -----------------------------------------------------
 
 # General information about the project.
-project = u'OME Data Model and File Formats'
+project = u'OME ' + model_version + u' Data Model and File Formats'
```

Comment:
With the last revert the project name is now mismatching with the index page. Should we drop `model_version` here too?